spec/models/like_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Should increment likes_counter' do
+    author = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    post = Post.new(author:, title: 'This is my last post', comments_count: 1, likes_count: 2)
```

Comment:
- [ ]  I could not run any of your tests due to some errors occurred in your tests for example here, while you were creating a post, the author did not have any value. Running the tests was raising syntax error because of that 
![image](https://user-images.githubusercontent.com/43877689/227714558-410d3146-d0d8-423b-9b18-4e631c28ad74.png)

Please make sure that you write tests for User, Post, Comment, Like model correctly, testing their customs methods from the above models

Reply:
hello @ishimwezachee , could you give me some guidance how to run the spec files, I could not run by writing rspec in my terminal

Reply:
> * [ ]   I could not run any of your tests due to some errors occurred in your tests for example here, while you were creating a post, the author did not have any value. Running the tests was raising syntax error because of that
>   ![image](https://user-images.githubusercontent.com/43877689/227714558-410d3146-d0d8-423b-9b18-4e631c28ad74.png)
> 
> Please make sure that you write tests for User, Post, Comment, Like model correctly, testing their customs methods from the above models

hello @ishimwezachee , could you give me some guidance how to run the spec files, I could not run by writing rspec in my terminal